web/src/containers/ApdStateKeyPerson.js

```diff
@@ -0,0 +1,250 @@
+                <TextField
+                  name={`apd-state-profile-costs${number}-fy${ffy}`}
+                  label={`FFY ${ffy}`}
+                  size="small"
+                  mask="currency"
+                  value={person.costs[ffy]}
+                  onChange={handleYearChange(ffy)}
+                />
+              </div>
+            ))}
+            <div className="ds-l-col--auto">
+              <FormLabel>Total</FormLabel>
+              <div className="form-value-input-aligned">
+                <Dollars>{personTotalCost(person)}</Dollars>
+              </div>
+            </div>
+          </div>
+        }
+      >
+        Yes
+      </Choice>
+      <Button
+        variation="primary"
+        className="ds-u-margin-top--4"
+        onClick={collapse}
+      >
+        Done
+      </Button>
+    </div>
+  );
+};
+PersonForm.propTypes = {
+  collapse: PropTypes.func.isRequired,
+  number: PropTypes.number.isRequired,
+  handleChange: PropTypes.func.isRequired,
+  handleYearChange: PropTypes.func.isRequired,
+  person: PropTypes.object.isRequired,
+  primary: PropTypes.bool.isRequired,
+  years: PropTypes.array.isRequired
+};
+
+class ApdStateKeyPerson extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { expanded: props.initialExpanded };
+  }
+
+  toggleExpanded = () => {
+    this.setState(prev => ({ expanded: !prev.expanded }));
+  };
+
+  render() {
+    const {
+      handleChange,
+      handleYearChange,
+      number,
+      person,
+      primary,
+      remove,
+      years
+    } = this.props;
+    const { expanded } = this.state;
+
+    return (
+      <Fragment>
+        <div className="visibility--screen">
+          {expanded ? (
+            <PersonForm
+              collapse={this.toggleExpanded}
+              handleChange={handleChange}
+              handleYearChange={handleYearChange}
+              number={number}
+              person={person}
+              primary={primary}
+              years={years}
+            />
+          ) : (
+            <Review
+              heading={`${number}. ${person.name}`}
+              editHref="#"
+              onEditClick={this.toggleExpanded}
+              editContent={
+                <div>
+                  <Button
+                    size="small"
+                    variation="transparent"
+                    onClick={this.toggleExpanded}
+                  >
+                    Edit
+                  </Button>
+                  {!primary && (
+                    <Fragment>
+                      |
+                      <Button
+                        size="small"
+                        variation="transparent"
+                        onClick={remove}
+                      >
+                        Remove
+                      </Button>
+                    </Fragment>
+                  )}
+                </div>
+              }
+            >
+              {person.position}
```

Comment:
I think it might make sense to make these into an unordered list, so that each item is its own `li`.

Reply:
YESSS, thank you!  When I was doing it, I was thinking to myself this was surely not the best approach, but a list just...  didn't...  occur to me.